### ISSUE TITLE
Use a currently-acceptable example skill?

### ISSUE THREAD

**Issue by wking** [instructlab/website#19]
The current example skill is poetry:

![Screenshot 2024-04-29 9 54 50 AM](https://github.com/instructlab/website/assets/209920/b79f9c52-fc55-4e29-96b0-a2cfc27d6708)

But [poetry is currently a topic to avoid][1].  Should we pivot to an example skill that has already landed (so users can go look up that exact skill for more details on it) and which we'd still accept today if it was submitted fresh.  Maybe [this one][2]?

[1]: https://github.com/instructlab/community/blob/364c086bf55875ec52e7deaf3ecf948231c699e6/docs/SKILLS_GUIDE.md#avoid-these-topics
[2]: https://github.com/instructlab/taxonomy/blob/4202260c47cf6c81fc910b2114bc8bf1f779f8ae/compositional_skills/extraction/technical_paper/abstract/reasoning/qna.yaml

**Comment by lhawthorn** [comment]
You are completely correct that the website graphic should not include an example that we have decided is not a topic we wish to use, as this can confuse our users and contributors. I am not sure what example would be the best one and am not in a good position to adjudicate thereon.

Pinging @akashgit for feedback.

**Comment by mariyageorge01** [comment]
This issue was raised on 29th April and the last comment was on 2nd May. 

Do we need to work on this issue? 

What should replace the current example skill?
